.github/workflows/node.install.yml

```diff
@@ -14,13 +14,13 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [18.x]
```

Comment:
Any specific reason not using the latest version (22.x) ?

Reply:
Mostly just because I was still running Node 18 on my desktop, but I updated to 22 and everything is fine so I've updated the CI to use 22 as well now.